src/components/policy/policy_external/src/sql_pt_representation.cc

```diff
@@ -649,10 +662,29 @@ bool SQLPTRepresentation::GatherFunctionalGroupings(
         }
       }
     }
+
+    rpcs.Reset();
+
     if (!rpcs_tbl.rpcs.is_initialized()) {
       rpcs_tbl.rpcs.set_to_null();
     }
-    rpcs.Reset();
+
+    // Collecting entities for disallowed_by_external_consent_entities_on/off
+    external_consent_entities.Bind(0, group_id);
+    while (external_consent_entities.Next()) {
+      policy_table::ExternalConsentEntity external_consent_entity(
+          external_consent_entities.GetInteger(0),
+          external_consent_entities.GetInteger(1));
+
+      policy_table::DisallowedByExternalConsentEntities&
+          external_consent_entities_container =
+              "ON" == external_consent_entities.GetString(2)
+                  ? *rpcs_tbl.disallowed_by_external_consent_entities_off
+                  : *rpcs_tbl.disallowed_by_external_consent_entities_off;
```

Comment:
@istoilovgithub typo? I suppose one of them should be on. Please mention to create something like UT to avoid such typos.